src/components/AudioFeatureSlider.tsx

```diff
@@ -0,0 +1,52 @@
+import React , { useState } from 'react';
+import Slider from '@material-ui/core/Slider';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core';
+
+const useStylesslider = makeStyles({
+    root: {
+      width: 300,
+    },
+  });
+export default function AudioFeatureSlider(props: {
+    feature_name: string,
+    feature_value: Number[],
+    delete: () => void;
+    giveFeaturesToPlaylist: (name:string, feature: Number[]) => void,
+}) {
+    const classes = useStylesslider();
+    //state for slider
+    const [SliderVal, setSliderVal] = useState(props.feature_value);
+
+    const handleChange = (event: any, value: number | number[]) => {
+        console.log('changing a value')
+        console.log(value)
+        setSliderVal(value as number[]);
```

Comment:
Has this been tested? Not 100% sure if TS casts `number` to `[number]`

Reply:
![image](https://user-images.githubusercontent.com/41666932/97541081-ae418980-1a18-11eb-8889-1b76dbf22d8a.png)

Reply:
The sliders get set as default to be a number[] when openned from dropdown so it can't be tested unless the slider turns into single value slider. But that removes the typecheck error since the slider object says it can return a number | number[]. 
is an if(typeof(value) === number) then return Number([value,value]) better here despite it being impossible?

Reply:
Should sliders be single valued? I thought were were opting for a dual slider (min and max) design

Reply:
I would recommend keeping type-confidence, and so instead opt for having a separate prop/callback/emitter when the slider changes, rather than changing the initial prop which used to contain the mix/max